### DISCUSSION TITLE
Need some Diagnostics Utils

### DISCUSSION THREAD

**Discussion by phoniclynx** [mKeRix/room-assistant#1112] [Ideas]
Been trying to set up my cluster for a while now but it's really hard to figure out what is going on as there is very little output for devices.

I think there needs to be some diagnostics regarding what BLE devices a room-assistant instance sees and how far away the iBeacon is from that particular instance. Not "just" when it sees it, but all the time. Something like this could be useful in the MQTT logs or even straight into Home Assistant, from each room-assistant instance at all times, from each device that is on the Allow list. 

Doing this would allow you to easily set up distances from each room to room so you can tell what settings you need to keep to have the reading be accurate.

This is handy I can see how far away it is and what threshold it should be so the readings don't overlap... this device isn't in the kitchen:
![image](https://user-images.githubusercontent.com/3883152/158934030-7cdfd300-e3a8-4f9a-a67a-b10fd10f0025.png)

This isn't handy but I can't see if one of the other room-assistant instances can see it at all so I can know if I need to adjust it:
![image](https://user-images.githubusercontent.com/3883152/158933984-5c5744c4-44a3-47c0-aa84-bbd0ba2ea82b.png)

**Comment by mKeRix** [comment]
Logging/exporting everything room-assistant sees at all times has garnered negative feedback in the past, because it seemed overwhelming. I do think it'd be nice to have an overview of all the measurements at all times though, irregardless of the allowlist. Could this maybe a be a good feature for #781?
